doc/releasenotes.rst

```diff
@@ -5,13 +5,23 @@ Release notes
 0.0 series
 ..........
 
+
+0.0.3 (2019-01-30)
+------------------
+
+* Import amici models and the petab data format automatically using
+  pypesto.PetabImporter.
+* Basic profiling routines.
```

Comment:
@paulstapor Could you shortly describe here what exactly exists already (unless "Basic profiling routines" is a good enough description already).